.github/workflows/integration.yml

```diff
@@ -0,0 +1,44 @@
+name: Integration Test opensearch-cli
+on:
+  push:
+    branches:
+      - main
+      - opensearch-*
+  pull_request:
+    branches:
+      - main
+      - opensearch-*
+
+jobs:
+  integ-test:
+    strategy:
+      matrix:
+        opensearch-version: [ 1.0.1 , 1.1.0, 1.2.4 ]
```

Comment:
Having some trouble with 1.3.  I am looking into it. 
For 2.0, i will create new PR since i have to checkout OpenSearch and assemble ( not released version).